src/Features/Core/Portable/ConvertToInterpolatedString/AbstractConvertPlaceholderToInterpolatedStringRefactoringProvider.cs

```diff
@@ -246,5 +271,19 @@ private class ConvertToInterpolatedStringCodeAction : CodeAction.DocumentChangeA
             {
             }
         }
+
+        private static class StringFormatArguments
+        {
+            public const string FormatArgumentName = "format";
+
+            public const string ArgsArgumentName = "args";
+
+            public static readonly Dictionary<int, string> ParamsArgumentNames = new Dictionary<int, string>
```

Comment:
:bulb: If you are just going to index into it (which the code currently is doing), we can benefit from both immutability and performance by changing this to `ImmutableArray<string>`:

```csharp
public static readonly ImmutableArray<string> ParamsArgumentNames =
  ImmutableArray.Create("", "arg0", "arg1", "arg2");
```